host/4/bullseye/amd64/powershell/powershell72/powershell72-appservice.Dockerfile

```diff
@@ -0,0 +1,68 @@
+# Build the runtime from source
+ARG HOST_VERSION=4.1.3
+FROM mcr.microsoft.com/dotnet/sdk:6.0.100 AS runtime-image
+ARG HOST_VERSION
+
+ENV PublishWithAspNetCoreTargetManifest=false
+
+# Build WebJobs.Script.WebHost from source
+RUN BUILD_NUMBER=$(echo ${HOST_VERSION} | cut -d'.' -f 3) && \
+    git clone --branch v${HOST_VERSION} https://github.com/Azure/azure-functions-host /src/azure-functions-host && \
+    cd /src/azure-functions-host && \
+    HOST_COMMIT=$(git rev-list -1 HEAD) && \
```

Comment:
there were recent changes made to ensure we do not have build commands to build host in every docker image. Please see https://github.com/Azure/azure-functions-docker/tree/dev/host/4/bullseye/amd64/node/node16 for example. Also, reach out to @CooperLink for assistance.

@CooperLink  - please add a readme on how to add new images following templates. Please open a  WI to track this devops board.

Reply:
We did not include powershell in that work yet because it was unsupported in v3.  Will be updating both of these images for v4. Working with Francisco on this and will introduce a WI for creating docs.

Reply:
Thanks! Lets make sure new images continue to follow templates pattern !

Reply:
Please ping for final sign-off once this comment is addressed as well.

Reply:
@CooperLink added the composite templates.